featuretools/entityset/entityset.py

```diff
@@ -1094,6 +1095,59 @@ def related_instances(self, start_entity_id, final_entity_id,
 
         return df
 
+    def plot(self, to_file=None):
+        """
+        Create a UML diagram-ish graph of the EntitySet.
+
+        Args:
+            to_file (str, optional) : Path to where the plot should be saved.
+                If set to None (as by default), the plot will not be saved.
+
+        Returns:
+            graphviz.Digraph : Graph object that can directly be displayed in
+                Jupyter notebooks.
+        """
+        if to_file:
+            # Explicitly cast to str in case a Path object was passed in
+            to_file = str(to_file)
+
+            split_path = to_file.split('.')
+            if len(split_path) < 2:
+                raise ValueError("Please use a file extension like '.pdf'" +
+                                 " so that the format can be infered")
+
+            format = split_path[-1]
+            valid_formats = graphviz.backend.FORMATS
+            if format not in valid_formats:
+                raise ValueError("Unknown format. Make sure your format is" +
+                                 " amongst the following: %s" % valid_formats)
+        else:
+            format = None
+
+        # Initialize a new directed graph
+        graph = graphviz.Digraph(self.id, format=format)
+
+        # Draw entities
+        for entity in self.entities:
+            variables_string = '\l'.join([var.id + ' : ' + var.dtype  # noqa: W605
+                                          for var in entity.variables])
+            label = '{%s|%s\l}' % (entity.id, variables_string)  # noqa: W605
+            graph.node(entity.id, shape='record', label=label)
+
+        # Draw relationships
+        for rel in self.relationships:
+            label = '%s -> %s' % (rel._child_variable_id, rel._parent_variable_id)
+            graph.edge(rel._parent_entity_id, rel._child_entity_id, label=label)
```

Comment:
i believe this edge is going in wrong direction. the arrow should point from the child to the parent. 

for example in the attached image the orders entity should points to the customers, rather than the other way around

![test](https://user-images.githubusercontent.com/371792/51942837-07b0fa80-23e6-11e9-8c52-5ab66d410b41.jpg)

Reply:
You're right regarding the direction. Also, [this page](https://developer.intacct.com/entity-relationship-diagrams/ap-payment/) suggests that there's a dedicated "one-to-many"-relationship arrowhead, which I also added with 1216fb4 and made all lines orthogonal. Furthermore, if the key is the same for both related entities, the key is now only displayed once.

The result now looks like this:
![download](https://user-images.githubusercontent.com/33159547/51993646-8a51bc80-24af-11e9-9f6d-cb8b24e7f0c1.png)

I'm not 100% satisfied with the aesthetics but I guess that'll have to do for now.